scripts/databricks_install.sh

```diff
@@ -0,0 +1,25 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+# ---------------------------------------------------------
+# This script installs Recommenders into Azure Databricks
+
+echo "Preparing Recommenders library file (egg)..."
+zip -r -q Recommenders.egg . -i *.py -x tests/\* scripts/\*
+
+echo "Uploading to databricks..."
+dbfs cp --overwrite Recommenders.egg dbfs:/FileStore/Recommenders.egg
+
+# Cluster id should be passed by the first argument
+# Cluster id can be found in the URL at https://<databricks-instance>/?o=<16-digit-number>#/setting/clusters/$CLUSTER_ID/configuration.
+CLUSTER_ID=$1
+
+echo "Installing the library onto databricks cluster $CLUSTER_ID..."
+databricks libraries install --cluster-id $CLUSTER_ID --egg dbfs:/FileStore/Recommenders.egg
+
+databricks libraries cluster-status --cluster-id $CLUSTER_ID
```

Comment:
this spits out a bunch of JSON, is it possible to parse this just to check the status of just the desired library?

Reply:
This was intentional to let users check not just Reco egg installation status but to show all the lib installed on the cluster. Parse and just show the lib name+status is possible. But if we want to show all the other libs status too, parsing gets complicated as the JSON has different keys /structure for different types of libs, e.g. pypi and egg. I rather keep the current form if it is okay.

Reply:
sure, just curious what options there are, but it looks like the cli can't filter or change the output format, so this is good to check them all I guess